### PR TITLE
dev-db/sqlitestudio: new package

### PR DESCRIPTION
**Pull request by rindeal**

### CODE REVIEW

File: dev-db/sqlitestudio/sqlitestudio-3.0.7.ebuild [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+	mkdir -p "${core_build_dir}" && cd "${core_build_dir}" || die
+	eqmake5 "${qmake_args[@]}" "${core_src_dir}"
+
+	## Plugins
+	mkdir -p "${plugins_build_dir}" && cd "${plugins_build_dir}" || die
+	eqmake5 "${qmake_args[@]}" "${plugins_src_dir}"
+}
+
+src_compile() {
+	cd "${core_build_dir}"		&& emake
```
Comment (monsieurp) [inline dev-db/sqlitestudio/sqlitestudio-3.0.7.ebuild]: Missing die.

File: dev-db/sqlitestudio/sqlitestudio-3.0.7.ebuild [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+	cd "${plugins_build_dir}"	&& emake
```
Comment (monsieurp) [inline dev-db/sqlitestudio/sqlitestudio-3.0.7.ebuild]: Ditto.

File: dev-db/sqlitestudio/sqlitestudio-3.0.7.ebuild [hunk 3]
```diff
@@ -0,0 +1,132 @@
...
+}
+
+src_install() {
+	cd "${core_build_dir}"		&& emake INSTALL_ROOT="${ED}" install
```
Comment (monsieurp) [inline dev-db/sqlitestudio/sqlitestudio-3.0.7.ebuild]: Ditto.

File: dev-db/sqlitestudio/sqlitestudio-3.0.7.ebuild [hunk 4]
```diff
@@ -0,0 +1,132 @@
...
+	cd "${plugins_build_dir}"	&& emake INSTALL_ROOT="${ED}" install
```
Comment (monsieurp) [inline dev-db/sqlitestudio/sqlitestudio-3.0.7.ebuild]: Ditto.

**Comment by monsieurp** [comment]
@gentoo/proxy-maint

**Comment by monsieurp** [comment]
Good to go after the minor errors pointed out are fixed.

**Comment by rindeal** [comment]
Ok, so should I `--amend` the commit or create another one?

**Comment by monsieurp** [comment]
Well it's up to you, really. You can commit on top of the previous commit, or amend, or commit and then squash to make it a single commit. As long as the errors are fixed, it doesn't matter. Pick your choice. :)

**Comment by rindeal** [comment]
> Well it's up to you, really. You can commit on top of the previous commit, or amend, or commit and then squash to make it a single commit. As long as the errors are fixed, it doesn't matter. Pick your choice. :)

Great, I went with the first variant.

**Pull request closed without merging**